examples/standard-https-with-cert-manager.yaml

```diff
@@ -0,0 +1,288 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: deployment
+---
+apiVersion: v1
+data:
+  ca.crt: ""
+  tls.crt: ""
+  tls.key: ""
+kind: Secret
+metadata:
+  name: my-ica-cert-and-key
+  namespace: deployment
+type: kubernetes.io/tls
+---
+apiVersion: v1
+data:
+  OS_PASSWORD: U3Q4cmxpbmdYKg==
+  OS_USERNAME: YWRtaW4=
+kind: Secret
+metadata:
+  name: system-endpoint
+  namespace: deployment
+stringData:
```

Comment:
You do not have OS_AUTH_TYPE in this block (but you do for your other blocks).  is it required?

Reply:
I used the respective templates for various configurations. Used the existing standard-https to create standard-https-with-cert-manager. Looks like the original was created without the OS_AUTH_TYPE in the standard config.